rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -3878,31 +3878,53 @@ void opcode_0160(const ScriptArguments& args, ScriptVec3 coord, const ScriptChan
     @arg arg3 Boolean true/false
     @arg blip Blip
 */
-void opcode_0161(const ScriptArguments& args, const ScriptVehicle vehicle, const ScriptBlipColour arg2, const ScriptBoolean arg3, ScriptBlip& blip) {
-    RW_UNIMPLEMENTED_OPCODE(0x0161);
-    RW_UNUSED(vehicle);
-    RW_UNUSED(arg2);
-    RW_UNUSED(arg3);
-    RW_UNUSED(blip);
-    RW_UNUSED(args);
+void opcode_0161(const ScriptArguments& args, const ScriptVehicle vehicle, const ScriptBlipColour colour, const ScriptBlipDisplay display, ScriptBlip& blip) {
+    auto& data = script::createObjectBlip(args, vehicle);
+    data.colour = colour;
+    /// @todo change ScriptBlipDisplay to BlipData::DisplayMode
+    data.display = static_cast <BlipData::DisplayMode> (display);
+    blip = &data;
 }
 
 /**
-    @brief %4d% = create_marker_above_actor %1d% color %2d% visibility %3d%
+    @brief ADD_BLIP_FOR_CHAR_OLD pedHandle colour display outblipHandle
```

Comment:
Is this change really useful? The comment on the left has been auto-generated. This change is fine if we did this renaming anyway.

However, I'm also worried *this* might even be the R* function name, which we tried to avoid for legal reasons.

Reply:
In fact, it is original R* function name.